### ISSUE TITLE
Enable GzipEncoderTests on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8777]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8777|
|Type|enhancement|
|Created|2016-08-21 06:35:18Z|
|Branch|https://github.com/twisted/twisted/tree/8777-rodrigc-gzipencoder-py3|

GzipEncoderTests need to be enabled on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8777 8777
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8777_rodrigc_gzipencoder_py3 8777-rodrigc-gzipencoder-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471761318564203 1471761318564203
changetime__1471776196530402 1471776196530402
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/515

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @rodrigc|
|-|-|

Reviewed by charmander https://github.com/twisted/twisted/pull/515#issuecomment-241250348

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset ab35b6ce88019c4436bb2542f8b144a28eae89cf
```
#!CommitTicketReference repository="" revision="ab35b6ce88019c4436bb2542f8b144a28eae89cf"
Merge pull request #515 from twisted/8777-rodrigc-gzipencoder-py3

Author: rodrigc
Reviewer: charmander
Fixes: #8777

Enable GzipEncoderTests and AllowedMethodTests on Python 3
```
